js/base/Exchange.js

```diff
@@ -2682,10 +2682,20 @@ module.exports = class Exchange {
     }
 
     parsePrecision (precision) {
+        /**
+         * @ignore
+         * @method
+         * @param {int} precision The number of digits to the right of the decimal
+         * @returns {string} a string number equal to 1e-precision
+         */
         if (precision === undefined) {
             return undefined;
         }
-        return '1e' + Precise.stringNeg (precision);
+        let parsedPrecision = '0.';
```

Comment:
@samgermain actually our Precise implement seems to support both formats 🤔 
![image](https://user-images.githubusercontent.com/43336371/224493313-0af88542-ca7f-4beb-bd98-69be0325475a.png)

can you pinpoint exactly where it fails?

Reply:
https://github.com/ccxt/ccxt/issues/15238#issuecomment-1445904549

Reply:
@kroitor do you foresee any issue if we get rid of the scientific notation?

Reply:
@carlosmiei @samgermain I don't foresee any issues with this change, but we're lacking tests here: https://github.com/ccxt/ccxt/blob/master/ts/src/test/Exchange/test.currency.ts